.gitignore

```diff
@@ -1,4 +1,5 @@
 composer.lock
 phpunit.xml
 vendor/
-libs/
\ No newline at end of file
+libs/
+**/.DS_Store
```

Comment:
what is this for?

Reply:
It a auto generated file. so moved  it in gitingore

Reply:
can we just not commit this file? instead of adding in gitignore?